src/applications/mhv/landing-page/mocks/api/index.js

```diff
@@ -12,6 +12,7 @@ const responses = {
   'GET /v0/user': user.defaultUser,
   'GET /v0/feature_toggles': featureToggles.generateFeatureToggles({}),
   'GET /my_health/v1/messaging/folders': folders.allFoldersWithUnreadMessages,
+  'GET /data/cms/vamc-ehr.json': {},
```

Comment:
i've been mocking the vamc-ehr.json response locally, which also requires a change to the local build config file. figured i'd just leave this in for others use, but happy to take out if requested as it is extraneous to this PR.

Reply:
also, maybe we should just address the underlying need for a call to vamc-ehr.json. it's a list of facilities — maintained in the CMS — that have converted to the cerner EHR. but we don't actually need that data on the landing page any more, b/c we aren't conditioning on the cerner-ness of a facility... might be worth a ticket...